content/references/rippled-api/public-rippled-methods/transaction-methods/tx.md

```diff
@@ -48,7 +48,11 @@ The request includes the following parameters:
 | `Field`       | Type    | Description                                        |
 |:--------------|:--------|:---------------------------------------------------|
 | `transaction` | String  | The 256-bit hash of the transaction, as hex.       |
-| `binary`      | Boolean | (Optional, defaults to false) If true, return transaction data and metadata as hex strings instead of JSON |
+| `binary`      | Boolean | _(Optional)_ If `true`, return transaction data and metadata as binary [serialized](serialization.html) to hexadecimal strings. If `false`, return transaction data and metadata as JSON. The default is `false`. |
+| `min_ledger`  | Number  | _(Optional)_ If provided, search for the transaction in ledger versions whose [ledger index][] is this value or higher. If provided, you should also specify `max_ledger`. [New in: rippled 1.5.0][] |
+| `max_ledger`  | Number  | _(Optional)_ If provided, search for the transaction in ledger versions whose [ledger index][] is this value or lower. If provided, you should also specify `min_ledger`. [New in: rippled 1.5.0][] |
```

Comment:
I don't think the description in these two rows is correct. "If provided, search for the transaction in ledger versions whose index is this value of higher". This isn't actually what happens, just as the "Caution" below notes. It searches the entire transaction database, regardless of what min_ledger and max_ledger are. It only uses min_ledger and max_ledger if it doesn't find the transaction, basically to report if there are any gaps in the requested range.

Reply:
I agree with @cjcobb23; it acts as he describes above. Nice comment @cjcobb23!

Reply:
I'm aware of this, but I struggled a lot with how to accurately and concisely characterize what these fields _actually_ do.

Something like, "If the transaction is not found, provides an [upper/lower] bound on the range of ledgers you would like the server to confirm that it searched" would be more accurate, but it takes some mental gymnastics to actually parse that. I'll keep thinking on how to describe it.